settings.py

```diff
@@ -25,4 +27,6 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.contenttypes",
+    "django.contrib.auth",
```

Comment:
actually default apps should be at the top of this list and then your apps and then third-party apps